CONTRIBUTING.md

```diff
@@ -50,3 +50,14 @@ try to consolidate identical literals so that they are not duplicated. For QStri
 cannot be merged.
 
 ([source](http://blog.qt.io/blog/2014/06/13/qt-weekly-13-qstringliteral/))
+
+# Internationalization do's and don'ts
+
+Do:
+* enable strings visible in the Mudlet GUI to be translateable
+* minimise use of HTML styling tags in strings to be translated
+
+Don't:
+* translate Mudlet API functions, events, or error messages
```

Comment:
I'd specify the `error messages` a bit further and make it `scripting error messages`. Because we may want to translate other error messages (like "can't connect" or other errors targeted at the actual user).